app/views/articles/_form.html.erb

```diff
@@ -1,26 +1,57 @@
-<%= form_with model: article do |form| %>
-  <div>
-    <%= form.label :title %><br>
-    <%= form.text_field :title %>
-    <% article.errors.full_messages_for(:title).each do |message| %>
-      <div><%= message %></div>
-    <% end %>
-  </div>
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Form</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  <head>
 
-  <div>
-    <%= form.label :body %><br>
-    <%= form.text_area :body %><br>
-    <% article.errors.full_messages_for(:body).each do |message| %>
-      <div><%= message %></div>
-    <% end %>
-  </div>
+  <body>      
+    <main class="container" id="main">
 
-  <div>
-    <%= form.label :status %><br>
-    <%= form.select :status, ['public', 'private', 'archived'], selected: 'public' %>
-  </div>
+      <%= form_with model: article do |form| %>
+        <div class="label-tittle-article">
+          <%= form.label :title %><br>
+          <%= form.text_field :title, required: true, class: "form-control" %>
+          <% article.errors.full_messages_for(:title).each do |message| %>
+            <div class="invalid-feedback">
+              <%= message %> 
+            </div>
+          <% end %>
+        </div>
 
-  <div>
-    <%= form.submit %>
-  </div>
-<% end %>
+        <div class="label-body-article">
+          <%= form.label :body %><br>
+          <%= form.text_area :body, required: true, class: "form-control" %><br>
+          <% article.errors.full_messages_for(:body).each do |message| %>
+            <div class="error-body-article">
+              <%= message %> 
+            </div>
+          <% end %>
+        </div>
+
+        <div class="label-select-article">
+          <%= form.label :status %><br>
+          <%= form.select :status, ['public', 'private', 'archived'], selected: 'public', id: "select-article" %>
+        </div>
+
+        <div class="label-submit-article">
+          <%= form.submit %>
+        </div>
+      <% end %>
+         
+    </main>
+
+    <footer>
```

Comment:
I see this repeated, create a partial or add the code to the main template

Reply:
Could you add that change in this PR?

Reply:
Sure, I'll add the change to this pull request.